_assets/js/templates/paginationTemplate.js

```diff
@@ -0,0 +1,80 @@
+// Imports:
+//= require ../utils/getOffsetParam.js
+//= require ../utils/setOffsetParam.js
+//= require ./nextButtonTemplate.js
+//= require ./previousButtonTemplate.js
+//= require ./ellipsisTemplate.js
+//= require ./paginationButtonTemplate.js
+
+const paginationTemplate = (resultsArr, urlParams) => {
+  let offsetInt = parseInt(urlParams.get("offset"));
```

Comment:
Does this exception handle if resultsArr is blank, or does not return?

Reply:
So in rendSearchPage.es6 there is an if statement that catches whether the results has any length or not. If not, we display a no results found paragraph element, otherwise we display the results. There is error handling in there as well, however nothing gets plopped in the DOM if there's an error. It just logs to the console.